### PR TITLE
Say cringe words when equipped with kitty ears give you 99 brain damage 

### PR DESCRIPTION
**Pull request by OneAsianTortoise**
## About The Pull Request
image
I want the guy to paypal me the therapy fee for the amount of cringe he manages to pack into three lines.

Saying cringe words now gives you _99_ brain damage **if you are wearing kitty ears.**

## Why It's Good For The Game
Duh

## Changelog

:cl:
add: save fulpstation
/:cl:

### CODE REVIEW

**Comment by OperativeLyn** [comment]
noob trap

**Comment by CheeseWizard2** [comment]
very based, but do you think it's obvious enough that saying the words is what causes the brain damage? i'd suggest maybe making it say something like "As you say the word "(x)", an overwhelming pain fills your head!"

**Comment by OneAsianTortoise** [comment]
my family and I were held at gunpoint to change the damage to 99.

(remember kid, BZ gas leak accident happens)

**Comment by OneAsianTortoise** [comment]
fuck up big time. i'll try to fix it

**Comment by OneAsianTortoise** [comment]
> Can you turn this into a regex

ok coder boy i did it u happy?

**Comment by githubuser4141** [comment]
> > noob trap
> 
> Yeah, as a new player I like to wear cat ear and say cringe word, totally.

Isn't 'nyah' part of the feline dialect lorewise? Why would saying it cause brain damage, and wouldn't this discourage roleplay?

**Comment by OneAsianTortoise** [comment]
> > > noob trap
> > 
> > Yeah, as a new player I like to wear cat ear and say cringe word, totally.
> 
> Isn't 'nyah' part of the feline dialect lorewise? Why would saying it cause brain damage, and wouldn't this discourage roleplay?

It affects kitty ears user, not felinid...???

**Comment by Horatio22** [comment]
> > > noob trap
> > 
> > 
> > Yeah, as a new player I like to wear cat ear and say cringe word, totally.
> 
> Isn't 'nyah' part of the feline dialect lorewise? Why would saying it cause brain damage, and wouldn't this discourage roleplay?

This is not the kind of RP we want to encourage. We removed felinids from roundstart races for a reason. If this is how someone wants to RP, there are quite handful of other servers that accommodate that. We aren't one of them.

File: fulp_modules/strings/kittyear/cringe_speech.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "cringe": [
+		"moff",
```
Comment (SmoSmoSmoSmok) [inline fulp_modules/strings/kittyear/cringe_speech.json:3]: Remove this from the list

Comment (SmoSmoSmoSmok) [reply fulp_modules/strings/kittyear/cringe_speech.json:3]: Immediately

**Comment by remove32** [comment]
gut reaction is that i dont like this in concept. generally, negative PRs are pretty uninteresting. open to counterpoint but if nobody can tell me why its good actually i'll be closing this req

**Comment by remove32** [comment]
got the go ahead to close this

**Pull request closed without merging**